examples/calculations/submit_md_using_config.py

```diff
@@ -18,8 +20,12 @@
 
 # All the other paramenters we want them from the config file
 # We want to pass it as a AiiDA data type for the provenance
+
+
 config = JanusConfigfile(
-    "/home/federica/aiida-mlip/tests/calculations/configs/config_janus_md.yaml"
+    Path("~/aiida-mlip/tests/calculations/configs/config_janus_md.yml")
```

Comment:
I think this probably ought to be the (expanded/resolved/absolute) relative path, since this assumes almost as much about the user's file structure as the original.

Reply:
No, you can get the path to this file, and use a path relative to that to get the absolute path to the config, completely independent to where the base `aiida-mlip` directory is.